addons/binding/org.openhab.binding.robonect/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openhab.binding</groupId>
+        <artifactId>pom</artifactId>
+        <version>2.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>org.openhab.binding.robonect</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
+
+    <name>Robonect Binding</name>
+    <packaging>eclipse-plugin</packaging>
+
+    <dependencies>
```

Comment:
another point I could not find when looking for it. Thank you for the link! Will change this of course